src/service.js

```diff
@@ -346,6 +346,7 @@ export function getParentWindowFrameElement(node, opt_topWin) {
  */
 export function getAmpdoc(nodeOrDoc) {
   if (nodeOrDoc.nodeType) {
+    // TODO: isConnected check on document?
```

Comment:
I believe `isConnected` is checked in the `ampdoc-impl`.

Reply:
This is for the case where you pass a `Node` to a service getter, not an `ampdoc`. Doing `node.ownerDocument` is not safe in cases where we inject a node into an FIE, so we need to wait till it's connected first.